elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/models/Column.java

```diff
@@ -153,13 +153,22 @@ private static String getTableSource(ColumnMeta meta, Column column) {
             }
         }
 
-        if (column.getColumnType() == REFERENCE) {
+        if (referencesAnotherModel(column.getExpression())) {
```

Comment:
tableSource is not something we want to infer. It must be provided by the developer in the HJSON file.  TableSource of NONE means no type ahead search should be provided by the API.  This change basically removes that option.